compiler/rustc_target/src/spec/armv7_unknown_trusty.rs

```diff
@@ -0,0 +1,31 @@
+use crate::spec::{Target, TargetOptions};
+
+use super::{crt_objects::LinkSelfContainedDefault, PanicStrategy, RelroLevel};
+
+pub fn target() -> Target {
+    Target {
+        // It's important we use "gnueabi" and not "musleabi" here. LLVM uses it
+        // to determine the calling convention and float ABI, and it doesn't
+        // support the "musleabi" value.
+        llvm_target: "armv7-unknown-linux-gnueabi".into(),
+        pointer_width: 32,
+        data_layout: "e-m:e-p:32:32-Fi8-i64:64-v128:64:128-a:0:32-n32-S64".into(),
+        arch: "arm".into(),
+        options: TargetOptions {
+            abi: "eabi".into(),
+            features: "+v7,+thumb2,+soft-float,-neon".into(),
+            max_atomic_width: Some(64),
+            mcount: "\u{1}mcount".into(),
+            os: "trusty".into(),
+            env: "musl".into(),
+            link_self_contained: LinkSelfContainedDefault::Musl,
```

Comment:
This seems to be missing for the aarch64 target. Is this intentional?

Reply:
From our initial testing it only seemed necessary for the 32 bit target. I tested adding it and it doesn't seem to cause any issues, so I've added it to the aarch64 target as well to keep them consistent.